src/components/OpenWith/OpenWith.js

```diff
@@ -112,16 +121,60 @@ class OpenWith extends PureComponent<Props, State> {
      * @return {void}
      */
     componentDidMount() {
-        const { fileId }: Props = this.props;
+        this.fetchOpenWithData();
+    }
+
+    /**
+     * Fetches file extension and Open With data.
+     *
+     * @return {void}
+     */
+    fetchOpenWithData() {
```

Comment:
Should this be named `fetchFileExtension`?

Reply:
It currently fetches the extension and the open with data